packages/Meshfree/src/DTK_DetailsMovingLeastSquaresOperatorImpl.hpp

```diff
@@ -253,16 +253,17 @@ struct MovingLeastSquaresOperatorImpl
         auto num_matrices =
             a.extent( 0 ) / ( size_polynomial_basis * size_polynomial_basis );
 
-        // TODO: right now, we hardcode the team size to 1. More information is
-        // available in the comments in SVDFunctor.
-        const int team_size = 1;
+        Kokkos::View<double **, DeviceType> aux( "aux", size_polynomial_basis,
+                                                 3 * num_matrices *
```

Comment:
Where does that 3 come from?

Reply:
Reading a commit message it seems it is because we need to store the matrices `E`, `U`, and `V`. Is that right? If so add a comment.

Reply:
That is correct, I added a comment.